packages/block-library/src/embed/deprecated.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Internal dependencies
+ */
+import { getEmbedSaveComponent } from './save';
+
+/**
+ * External dependencies
+ */
+import classnames from 'classnames/dedupe';
```

Comment:
Interesting, not a typo :D https://github.com/JedWatson/classnames#alternate-dedupe-version

"This version is slower (about 5x) so it is offered as an opt-in." Both appear to be in use in Gutenberg.